x-pack/plugins/siem/public/containers/errors/index.tsx

```diff
@@ -32,3 +32,9 @@ export const errorLink = onError(({ graphQLErrors, networkError }) => {
     );
   }
 });
+
+export const refetchOnErrorLink = onError(({ graphQLErrors, networkError, operation, forward }) => {
```

Comment:
I think we should only retry on 503 network error and we should rename the function to reTryOneTimeOnErrorLink
```
export const reTryOneTimeOnErrorLink = onError(({ networkError, operation, forward }) => {
  if (networkError != null) {
    const statusCode = get('statusCode', networkError);
    if (statusCode != null && statusCode === 503) {
      return forward(operation);
    }
  }
});
```

Reply:
Add unit test just make sure, we only retry for 503;

Reply:
Good point! Updated the way of handling this to https://github.com/elastic/ingest-dev/issues/522, unit test added as well.

Reply:
I'm wondering, would this respect the Retry-After header? I worry that if the 503 is due to load, retrying will make it harder for the server to recover.